client/store/orders.js

```diff
@@ -46,13 +47,26 @@ export const newOrderForGuestThunk = (order) => {
   }
 }
 
+export const newOrderForUserThunk = (order) => {
+  return async (dispatch) => {
+    try{
+      const {data: newOrder} = await axios.post('/api/user/orders', order)
+      dispatch(newOrderForUserAction(newOrder))
+    } catch (error) {
+      console.error(error);
+    }
+  }
+}
+
 // reducer
 export default (state = initialState, action) => {
   switch (action.type) {
     case GOT_ORDERS:
       return action.orders
     case NEW_ORDER_FOR_GUEST:
       return [...state, action.order]
+    case NEW_ORDER_FOR_USER:
```

Comment:
Do we need two different actions? the resulting update to the store is the same, right?